src/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -78,7 +79,7 @@
     abstract public SamReaderFactory referenceSequence(File referenceSequence);
 
     /** Sets the specified reference sequence * */
-    abstract public SamReaderFactory referenceSource(ReferenceSource referenceSequence);
+    abstract public SamReaderFactory referenceSource(CRAMReferenceSource referenceSequence);
```

Comment:
It seems awkward that this abstract class `SamReaderFactory` should have CRAM-specific parameters/return values like this -- though I suppose that was always the case, just disguised under the former, more generic name.